CONTRIBUTING.md

```diff
@@ -20,6 +20,7 @@ Testing:
 * New examples must run! You *must* test your code on either a robot or in
   simulation. If there's something broken in RobotPy, file an issue to get it
   fixed
+  * To test a vision file, run `python -m cscore vision.py` in the directory of the vision file
```

Comment:
We should link to the docs instead: https://robotpy.readthedocs.io/en/stable/vision/other.html#vision-other-runcustom

Reply:
Done:
https://github.com/robotpy/examples/pull/104/commits/30aef8b6621345eae6a9752ea6471868ef240e9d